SU2_CFD/src/solvers/CIncEulerSolver.cpp

```diff
@@ -2310,6 +2310,30 @@ void CIncEulerSolver::BC_Inlet(CGeometry *geometry, CSolver **solver_container,
         break;
     }
 
+    /* Loop over markers and check if the vertex is on another marker. If we have found 
+       that a vertex is on a viscous wall, we impose the no-slip condition and the pressure 
+       from the interior.
+    */
+
+   /*--- loop over markers in geometry ---*/
+    for (auto iMarker = 0; iMarker < config->GetnMarker_All(); iMarker++) {
+      /*--- check it is a viscous wall ---*/
+      if( config->GetViscous_Wall(iMarker)) {
```

Comment:
For CEulerSolver, we need to do this in the same manner for velocity and pressure, for both total_pressure and massflow boundary type?

Reply:
It is done already, since for compressible only the flow direction can be specified. Supersonic inlet is the only one that may need this treatment.
It is probably fine to leave temperature alone even in isothermal boundaries, having 0 velocity should avoid wrong contributions to the energy equation, we should run the "profile test" on temperature.

Reply:
OK, thanks. The failed regressions are all directly related to the change in residuals due to this modification. I can update it if there are no other changes in the code necessary for this PR.